package-manager/README.md

```diff
@@ -0,0 +1,159 @@
+# Posit Package Manager Container Image
+
+This container image provides [Posit Package Manager](https://docs.posit.co/rspm/) (PPM), a repository management server that organizes and centralizes R and Python packages across teams, departments, or organizations.
+
+> [!IMPORTANT]
+> This image is under active development and testing and is not yet supported by Posit.
+>
+> Please see [rstudio-package-manager image](https://github.com/rstudio/rstudio-docker-products/tree/main/package-manager) in `rstudio/rstudio-docker-products` for the officially supported image.
+
+## Quick Start
+
+```bash
+docker run -d \
+  --name package-manager \
+  -p 4242:4242 \
+  -v /path/to/license.lic:/etc/rstudio-pm/license.lic \
+  posit/package-manager:2025.09.0-7
+```
+
+Access Package Manager at `http://localhost:4242`.
+
+## Image Variants
+
+Two variants are available:
+
+| Variant          | Description                                               |
+|------------------|-----------------------------------------------------------|
+| `std` (Standard) | Includes R and Python for package building and validation |
+| `min` (Minimal)  | Package Manager only, smaller image size                  |
+
+## Image Tags
+
+Images are published to:
+- Docker Hub: `docker.io/posit/package-manager`
+- GitHub Container Registry: `ghcr.io/posit-dev/package-manager`
+
+Tag formats:
+- `2025.09.0-7` - Full version (standard variant, Ubuntu 22.04)
+- `2025.09.0-7-ubuntu-22.04-std` - Explicit OS and variant
+- `2025.09.0-7-ubuntu-24.04-min` - Ubuntu 24.04 minimal variant
+- `latest` - Latest stable release (standard variant, Ubuntu 22.04)
+
+## Configuration
+
+### License Activation
+
+A valid license is required. Choose one method:
+
+**Option 1: License File (Recommended)**
+```bash
+docker run -v /path/to/license.lic:/etc/rstudio-pm/license.lic ...
+```
+
+**Option 2: Floating License Server**
+```bash
+docker run -e PPM_LICENSE_SERVER="http://license-server:8989" ...
+```
```

Comment:
At some point I would like to learn if this is in widespread use and where. If it is heavily used in something like k8s, we may want to consider making a companion chart for it eventually.